vips/vip-491/bsctestnet-2.ts

```diff
@@ -0,0 +1,289 @@
+          signature: "transfer(address,uint256)",
+          params: [sepolia.VTREASURY, vTokensRemaining],
+          dstChainId: LzChainId.sepolia,
+        };
+      })(),
+
+      {
+        target: VsUSDe_CORE,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.05", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+      // Add USDe Market
+      // oracle config
+      {
+        target: sepolia.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            sUSDe,
+            [sUSDe_ERC4626ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      // Market configurations
+      {
+        target: VUSDe_CORE,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: USDe,
+        signature: "faucet(uint256)",
+        params: [USDe_INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: USDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: USDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, USDe_INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VUSDe_CORE,
+            parseUnits("0.72", 18), // CF
+            parseUnits("0.75", 18), // LT
+            USDe_INITIAL_SUPPLY, // initial supply
+            sepolia.NORMAL_TIMELOCK,
+            parseUnits("30000000", 18), // supply cap
+            parseUnits("25000000", 18), // borrow cap
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: VUSDe_CORE,
+        signature: "transfer(address,uint256)",
+        params: [ethers.constants.AddressZero, parseUnits("100", 8)], // around $100
+        dstChainId: LzChainId.sepolia,
+      },
+      (() => {
+        const vTokensMinted = convertAmountToVTokens(USDe_INITIAL_SUPPLY, parseUnits("1", 28));
+        const vTokensRemaining = vTokensMinted.sub(parseUnits("100", 8));
+        return {
+          target: VUSDe_CORE,
+          signature: "transfer(address,uint256)",
+          params: [sepolia.VTREASURY, vTokensRemaining],
+          dstChainId: LzChainId.sepolia,
+        };
+      })(),
+      {
+        target: VUSDe_CORE,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.05", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+      // configure converters for sUSDe and USDe
+      // Conversion config for sUSDe
```

Comment:
I think we don't need the configuration of conversions for sUSDe, because they should already be in place (there is a market for sUSDe already on Ethereum). Only the configuration for the conversions of USDe is new

Reply:
agreed ! b105546d8735819c17260cf0f340fb8b253c6585